en_US.ini

```diff
@@ -598,10 +598,10 @@ Force real clock sync (slower, less lag) = Force real clock sync (slower, less l
 General = General
 Help the PPSSPP team = Help the PPSSPP team
 Host (bugs, less lag) = Host (bugs, less lag)
-I/O on thread (experimental) = I/O on thread (experimental)
+I/O on thread (experimental) = I/O on thread
 IO timing method = I/O timing method
 MMDDYYYY = MMDDYYYY
-Multithreaded (experimental) = Multithreaded (experimental)
+Multithreaded (experimental) = Multithreaded (glitches)
```

Comment:
Yes, some games get framerate problems or just have minor graphics issues in certain places.  Other games completely break.

-[Unknown]